docs/howto/connection/index.rst

```diff
@@ -118,62 +118,53 @@ The JSON format contains an object where the key contains the connection ID and
 .. code-block:: json
 
     {
-        "CONN_A": {
-            "conn_type": "mysql",
-            "host": "mysql",
-            "login": "root",
-            "password": "plainpassword",
-            "schema": "airflow",
-            "port": null,
-            "extra": null,
-            "is_encrypted": false,
-            "is_extra_encrypted": false
-        },
-        "CONN_B": {
-            "conn_type": "druid",
-            "host": "druid-broker",
-            "login": null,
-            "password": null,
-            "schema": null,
-            "port": 8082,
-            "extra": "{\"endpoint\": \"druid/v2/sql\"}",
-            "is_encrypted": false,
-            "is_extra_encrypted": false
-        }
+      "airflow_db": {
+        "conn_type": "mysql",
+        "host": "mysql",
+        "login": "root",
+        "password": "plainpassword",
+        "schema": "airflow",
+        "port": null,
+        "extra": null
+      },
+      "druid_broker_default": {
+        "conn_type": "druid",
+        "host": "druid-broker",
+        "login": null,
+        "password": null,
+        "schema": null,
+        "port": 8082,
+        "extra": "{\"endpoint\": \"druid/v2/sql\"}"
+      }
     }
 
 The YAML file structure is similar to that of a JSON. The key-value pair of connection ID and the definitions of one or more connections. In this format, the connection is defined as a YAML object. The following is a sample YAML file.
 
 .. code-block:: yaml
 
-    CONN_A:
+    airflow_db:
       conn_type: mysql
-      extra:
+      extra: null
       host: mysql
-      is_encrypted: false
-      is_extra_encrypted: false
       login: root
       password: plainpassword
-      port:
+      port: null
       schema: airflow
-
-    CONN_B:
+    druid_broker_default:
       conn_type: druid
       extra: '{"endpoint": "druid/v2/sql"}'
       host: druid-broker
-      is_encrypted: false
```

Comment:
These fields do not exist in the output file.

Reply:
Ah! missed that. I had removed both of those fields in later version. Thanks 👍